frontend/pages/admin/families/[id]/entities.vue

```diff
@@ -21,6 +21,7 @@
           :on-save-callback="onSave"
           kind-name="entité"
           kind="entity"
+          :categories="categories"
```

Comment:
Place it with other `:` parameters (before the kind)

Reply:
I do not see the point, but sure if you prefer